src/packages.ts

```diff
@@ -94,11 +87,35 @@ export class PackageManager {
 
         pkg.tmpFile = tmpResult;
 
-        const result = await downloadFile(url, pkg, logger, status, proxy);
+        const result = await downloadFile(latestSnootyReleaseUrl, pkg, logger, status, proxy);
         logger.appendLine(' Done!');
 
         return result;
     }
+
+    public async getLatestParserUrl(platform: string, logger: Logger) {
+        const githubUrl = `https://github.com/mongodb/snooty-parser/releases/latest`;
```

Comment:
What happens if GitHub returns an error code, i.e. if there's a transient GitHub issue or if the writer is not connected to the internet/has a spotty connection?